onnxruntime/python/tools/transformers/models/whisper/whisper_helper.py

```diff
@@ -71,7 +71,7 @@ def get_onnx_path(
 
     @staticmethod
     def load_model(
-        model_name_or_path: str, cache_dir: str, device: torch.device, merge_encoder_and_decoder_init: bool = True
+        model_name_or_path: str, cache_dir: str, device: torch.device, merge_encoder_and_decoder_init: bool = True, state_dict: str = None
```

Comment:
Change `state_dict` to `state_dict_path` to match the other parameter names since it is a file path